fe/src/project/AspNetCore/areas/Home/components/NewsCatalog/NewsCatalog.js

```diff
@@ -0,0 +1,65 @@
+import React, {useState} from "react";
```

Comment:
**suggestion:** I'd get rid of it completely. Let's use the existing CarouselGrid and maybe extend the card-content a little bit to contain the information category and date in CarouselGrid.js.